src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -155,7 +156,9 @@ export class MeetingManager implements AudioVideoObserver {
       try {
         await this.audioVideo.chooseVideoInputDevice(null);
         await this.audioVideo.chooseAudioInputDevice(null);
-        await this.audioVideo.chooseAudioOutputDevice(null);
+        if (supportsSetSinkId()) {
+          await this.audioVideo.chooseAudioOutputDevice(null);
+        }
```

Comment:
I think this change is incorrect — don't we need the `null` path through to do the initial bind?